### PR TITLE
Fix Broken Links

### PR DESCRIPTION
**Pull request by justindhillon**
There are multiple broken links in this project. Here are some of the ones I have fixed:

https://www.cs.ubc.ca/~acm-web/practice/ --> https://www.cs.ubc.ca/students/undergrad/life/acm

https://ubccsss.org/print --> https://ubccsss.org/cube/printer/

https://www.ams.ubc.ca/healthplan --> https://www.ams.ubc.ca/support-services/health-dental/

These broken links where found with [link-inspector](https://github.com/justindhillon/link-inspector)

### CODE REVIEW

File: themes/hugo-bootstrap-5/exampleSite/content/post/migrate-from-jekyll.md
Comment (kewbish) [inline themes/hugo-bootstrap-5/exampleSite/content/post/migrate-from-jekyll.md:1]: We don't need to remove the dead links here, as this is example code copied directly from the template we were basing our theme off of. If you'd like, you can just delete `exampleSite/`.

File: themes/hugo-bootstrap-5/layouts/partials/event/add-to-calendar.html
Comment (kewbish) [inline themes/hugo-bootstrap-5/layouts/partials/event/add-to-calendar.html:1]: Please don't remove this link, it's functional and in use on our event pages.

File: themes/hugo-bootstrap-5/layouts/tcf2023/tcf-main.html [hunk 1]
```diff
@@ -85,15 +85,6 @@ <h1 class="display-1 fw-bold">Technical Career Fair 2023</h1>
                     >+ Google Calendar</a
                   >
                 </li>
-                <li>
```
Comment (kewbish) [inline themes/hugo-bootstrap-5/layouts/tcf2023/tcf-main.html:88]: See above.

**Review by kewbish — COMMENTED** [review]
Thanks for running your tool on the repo, this is very helpful!

File: content/cube/printer.md [hunk 2]
```diff
@@ -13,4 +13,4 @@ cheaper than UBC printers!
 - 10¢ per color page
 
 You may print by either using the public computer in the Cube, or by connecting
-to [Google Cloud Print](https://ubccsss.org/print).
+to [Google Cloud Print](https://ubccsss.org/cube/printer/).
```
[the remark this answers is not in the document]
Comment (justindhillon) [reply content/cube/printer.md:16]: Sounds great! See #516 for the pr

File: themes/hugo-bootstrap-5/exampleSite/content/post/migrate-from-jekyll.md
Comment (justindhillon) [reply themes/hugo-bootstrap-5/exampleSite/content/post/migrate-from-jekyll.md:1]: Sounds good. See #517 for the pr

File: themes/hugo-bootstrap-5/layouts/partials/event/add-to-calendar.html
Comment (justindhillon) [reply themes/hugo-bootstrap-5/layouts/partials/event/add-to-calendar.html:1]: Good to know, it is now reverted.

File: themes/hugo-bootstrap-5/layouts/tcf2023/tcf-main.html (hunk 1 above)
Comment (justindhillon) [reply themes/hugo-bootstrap-5/layouts/tcf2023/tcf-main.html]: Good to know, it is now reverted.

File: content/about/clubs.md [hunk 3]
```diff
@@ -25,7 +25,7 @@ The following UBC CS-related clubs are partners of the CSSS. Click on each club
     {{< club title="NWPlus" href="https://nwplus.io/" src="/files/nwplus.jpg" description="A student-led club supporting aspiring programmers and designers, based out of UBC." facebook="https://www.facebook.com/nwplusubc" instagram="https://www.instagram.com/nwplusubc" blog="https://medium.com/nwplusubc" linktree="https://linktr.ee/nwplusubc" >}}
   </div>
   <div class="col-lg-6 col-xl-4 mb-4 d-flex align-items-stretch">
-    {{< club title="UBC Game Dev" href="https://www.ubcgamedev.com/" src="/files/ubcgamedev.png" description="A student run video game development club aiming to give students a hands-on approach in making video games." discord="https://discord.io/ubcgamedev" facebook="https://www.facebook.com/ubcgamedev/" instagram="https://www.instagram.com/ubcgamedev/?hl=en" >}}
+    {{< club title="UBC Game Dev" href="https://www.ubcgamedev.com/" src="/files/ubcgamedev.png" description="A student run video game development club aiming to give students a hands-on approach in making video games." discord="https://discord.com/invite/xF3WbYDubF" facebook="https://www.facebook.com/ubcgamedev/" instagram="https://www.instagram.com/ubcgamedev/?hl=en" >}}
```
[the remark this answers is not in the document]
Comment (justindhillon) [reply content/about/clubs.md]: Sorry about that. It has now been changed.

File: content/about/clubs.md [hunk 4]
```diff
@@ -13,7 +13,7 @@ The following UBC CS-related clubs are partners of the CSSS. Click on each club
 	{{< club title="UBC CTF (Maple Bacon)" href="https://maplebacon.org" src="/files/ubcctf.png" id="maple-bacon" description="A CTF Team at the University of British Columbia." blog="https://maplebacon.org/blog/" discord="https://discord.gg/keeTZsmfVA" >}}
   </div>
   <div class="col-lg-6 col-xl-4 mb-4 d-flex align-items-stretch">
-    {{< club title="ACM" href="https://www.cs.ubc.ca/~acm-web/practice/" src="/files/ubcacm.png" description="A UBC Competitive Programming Team." discord="https://discord.gg/jfZHF7J" facebook="https://www.facebook.com/groups/831817450248984/about/" >}}
+    {{< club title="ACM" href="https://www.cs.ubc.ca/students/undergrad/life/acm" src="/files/ubcacm.png" description="A UBC Competitive Programming Team." discord="https://discord.gg/jfZHF7J" facebook="https://www.facebook.com/groups/831817450248984/about/" >}}
```
[the remark this answers is not in the document]
Comment (justindhillon) [reply content/about/clubs.md]: Sorry about that. It has now been changed.

**Review by kewbish — APPROVED** [review]

**Comment by kewbish** [comment]
Could you rebase on master to address the merge conflict? I'll merge after that.

**Pull request merged**